plugins/parcelport/CMakeLists.txt

```diff
@@ -15,6 +15,9 @@ if(HPX_WITH_NETWORKING)
     verbs
     mpi
     tcp)
+elseif(HPX_MPI_WITH_FUTURES)
+  set(parcelport_plugins ${parcelport_plugins}
+    mpi)
```

Comment:
Is this still needed? The MPI parcelport is anyway guarded by `HPX_WITH_NETWORKING`.

Reply:
You are right, this is not needed anymore.